assets/javascripts/discourse/initializers/nested-posts.js

```diff
@@ -0,0 +1,71 @@
+import { withPluginApi } from "discourse/lib/plugin-api";
+import { transformBasicPost } from "discourse/lib/transform-post";
+import { postTransformCallbacks } from "discourse/widgets/post-stream";
+
+const PLUGIN_ID = "discourse-nested-posts";
+
+export default {
+  name: PLUGIN_ID,
+
+  initialize(container) {
+    const nestedPosts = container.lookup("service:nested-posts");
+
+    if (nestedPosts.isEnabled) {
+      withPluginApi("1.3.0", (api) => {
```

Comment:
do we need to be backward compatible? Maybe we can take latest version 1.6.0? https://github.com/discourse/discourse/blob/main/app/assets/javascripts/discourse/app/lib/plugin-api.js#L119

Reply:
Bumped. Thanks!